com.sap.cloud.lm.sl.cf.database/src/main/java/com/sap/cloud/lm/sl/cf/database/migration/executor/DatabaseTableMigrationExecutor.java

```diff
@@ -24,11 +24,11 @@
     @Override
     public void executeMigrationInternal(String databaseTable) throws SQLException {
         logger.info("Migrating table \"{}\"...", databaseTable);
-        DatabaseTableMetadata sourceTableMetadata = extractTableMetadataFromSourceDatabase(databaseTable);
+        DatabaseTableData sourceTableMetadata = extractTableMetadataFromSourceDatabase(databaseTable);
```

Comment:
You've renamed the type, but not some of the variables/methods. Rename them for consistency.